### PR TITLE
Add sample with dockerized HZ and dockerized Kafka

### PR DESCRIPTION
**Pull request by ldziedziul**
This PR adds a sample running against standalone Hazelcast and Kafka (using docker-compose)

### CODE REVIEW

File: jet/kafka/src/main/java/com/hazelcast/samples/jet/kafka/KafkaSourceWithStandaloneHZAndKafka.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    private static final String SINK_NAME = "sink";
+    private static final String INTERNAL_DOCKER_BROKER_ADDRESS = "broker:29092";
+    private static final String EXTERNAL_DOCKER_BROKER_ADDRESS = "localhost:9092";
+
+    private TopicUtil topicUtil;
+
+    private static Pipeline buildPipeline() {
+        Pipeline p = Pipeline.create();
+        //we need to use an internal broker address as the pipeline will be executed in the HZ container
+        p.readFrom(KafkaSources.kafka(props(
```
[the remark this answers is not in the document]
Comment (ldziedziul) [reply jet/kafka/src/main/java/com/hazelcast/samples/jet/kafka/KafkaSourceWithStandaloneHZAndKafka.java:62]: This is a sample so I think it should be as straightforward as possible to help newcomers find the right pieces ;)

File: jet/kafka/src/main/java/com/hazelcast/samples/jet/kafka/KafkaSourceWithStandaloneHZAndKafka.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            Pipeline p = buildPipeline();
+
+            long start = System.nanoTime();
+            Job job = jet.newJob(p);
+            while (true) {
+                int mapSize = sinkMap.size();
+                System.out.format("Received %d entries in %d milliseconds.%n",
+                        mapSize, NANOSECONDS.toMillis(System.nanoTime() - start));
+                if (mapSize == MESSAGE_COUNT_PER_TOPIC * 2) {
+                    job.cancel();
```
[the remark this answers is not in the document]
Comment (ldziedziul) [reply jet/kafka/src/main/java/com/hazelcast/samples/jet/kafka/KafkaSourceWithStandaloneHZAndKafka.java:96]: I see no scary errors in the logs:

image

I would keep it as simple as possible.

**Review by TomaszGaweda — APPROVED** [review]

**Comment by TomaszGaweda** [comment]
Looks good :)

File: jet/kafka/src/main/java/com/hazelcast/samples/jet/kafka/KafkaSourceWithStandaloneHZAndKafka.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (TomaszGaweda) [reply jet/kafka/src/main/java/com/hazelcast/samples/jet/kafka/KafkaSourceWithStandaloneHZAndKafka.java:97]: oh, ok, you are logging it on fine level ;) Mislooked that, I agree on keeping it simple then

**Pull request merged**